packages/botonic-react/src/components/message/message-image.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext } from 'react'
+
+import { WEBCHAT } from '../../constants'
+import { WebchatContext } from '../../contexts'
+import { resolveImage } from '../../util/environment'
+import { BotMessageImageContainer } from './styles'
+
+interface MessageImageProps {
+  imagestyle: Record<string, unknown>
```

Comment:
camelCase: imageStyle?

Reply:
No the correct prop is imagestyle. There is an old [PR#1063](https://github.com/hubtype/botonic/pull/1063) in the description of this PR that says it is better to use imagestyle. 
There is also a code that makes a log saying that legacy is imageStyle and the new imagestyle (src/util/logs.js).